packages/terra-demographics-banner/tests/nightwatch/demographics-banner-spec.js

```diff
@@ -0,0 +1,95 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
The tests are actually covered here. https://github.com/cerner/terra-core/pull/280/files/37edad6708a2e17ae4d7f86f5cf210e096ad2e8b#diff-6cdd46400fbcd359833cabe8b42eb5bdR61

Since this same file runs across all different screen sizes, it's hard to write a separate file for small viewport tests.

If you like, I can add more comments in this file to indicate that.